custom_cards/custom_card_ristou_person/custom_card_ristou_person.yaml

```diff
@@ -0,0 +1,386 @@
+                              }
+                            } else{
+                              return "rgba(var(--color-green),1)";
+                            }
+                          }
+                        ]]]
+                    - display: >
+                        [[[
+                              if (variables.ulm_custom_card_ristou_use_badge) {
+                                  return "block";
+                              } else {
+                                  return "none";
+                              }
+                        ]]]
+              custom_fields:
+                notification: >
+                  [[[
+                    var zones = variables.ulm_custom_card_ristou_zones;
+                    var person_location = entity.state;
+                    var driving_state = "off"
+                    var icon = "mdi:help-circle"
+                    if (variables.ulm_custom_card_ristou_person_driving_entity !== "") {
+                      driving_state = states[variables.ulm_custom_card_ristou_person_driving_entity].state;
+                    }
+                    if (driving_state === "on" || driving_state === "true") {
+                      icon = 'mdi:car';
+                    } else {
+                      if (person_location !== 'home'){
+                        for (const item of zones){
+                          if (person_location == states[item]?.attributes?.friendly_name){
+                            icon = (states[item].attributes.icon !== null) ? states[item].attributes.icon : 'mdi:help-circle';
+                          } else if (person_location == 'not_home'){
+                            icon = 'mdi:home-minus';
+                          }
+                        }
+                      } else{
+                        icon = 'mdi:home-variant';
+                      }
+                    }
+                    return '<ha-icon icon="' + icon + '" style="width: 10px; height: 10px; color: var(--primary-background-color);"></ha-icon>';
+                  ]]]
+          #####################
+          ### LABEL + STATE ###
+          #####################
+          item2:
+            card:
+              type: "custom:button-card"
+              entity: "[[[ return entity.entity_id ]]]"
+              name: "[[[ return variables.ulm_custom_card_ristou_name ]]]"
+              label: >-
+                [[[
+                  var location = entity.state
+                  var driving_state = "off"
+                  if (variables.ulm_custom_card_ristou_person_driving_entity !== "") {
+                    driving_state = states[variables.ulm_custom_card_ristou_person_driving_entity].state;
+                  }
+                  if (driving_state === "on" || driving_state === "true") {
+                    return variables.ulm_custom_card_ristou_person_driving;
+                  } else {
+                    if (location == "unavailable"){
```

Comment:
`ulm_unavailable` is deprecated and now part of the translation engine. Also home and not_home states are available. See example from the `card_person`:
```yaml
label: >
    [[[
      if (variables.ulm_address){
        return states[variables.ulm_address].state
      } else {
        let state = states[variables.ulm_card_person_entity].state;
        let option = ["home", "not_home", "unavailable", "unknown"]
        return (option.includes(state)) ? variables.ulm_translation_state : state
      }
```